aineko/cli/run.py

```diff
@@ -6,12 +6,29 @@
 import traceback
 from typing import Optional
 
+import click
+
 from aineko.core.runner import Runner
 
 logger = logging.getLogger(__name__)
 
 
-def main(
+@click.command()
+@click.argument("pipeline-config-file")
+@click.option(
+    "--pipeline-name",
+    "-p",
+    default=None,
+    help="Name of the pipeline to run (optional).",
```

Comment:
```suggestion
    help="Name of the pipeline to run.",
```
It's already optional, so I don't think it's helpful to mention it again in the help.